bedrock/firefox/views.py

```diff
@@ -43,7 +43,7 @@
     'release',
     'beta',
     'alpha',
-    # 'nightly',  # soon
+    'nightly',
```

Comment:
Do we need to add `aurora` here too? I still see traffic with that parameter in GA

Reply:
When I append `?channel=aurora` to the URL it shows a download for `?product=firefox-aurora-latest-ssl` which 404's, so I don't think we should but it's possible I am misunderstanding.

Reply:
*The 404 behavior happens when `aurora` is included in this array. 

If `aurora` is what dev edition used to be, we might need to redirect those users to `alpha` or ask that that query parameter is replaced with `alpha`?

Reply:
Perhaps we chould be treat `alpha` and `aurora` as the same? I believe we should be able to do that without changing or rewriting the query parameter itself, if we handled it in the view logic.